client/src/App.js

```diff
@@ -1,10 +1,18 @@
 import { useState, useEffect } from 'react'
-import { Route, Switch, BrowserRouter as Router } from 'react-router-dom'
+import { Route, Switch, BrowserRouter as Router, Link } from 'react-router-dom'
+
 import { loadFromLocal, saveToLocal } from './library/localStorage'
+import { deleteItem, filterActiveIngredients, toggleIngredient } from './library/ingredientsHelpers'
```

Comment:
Uhh I can see refactoring <3

Reply:
Haha, just some minor refactoring. I will leave it till the end, so I can see the patterns in my whole code. :)